erpnext/vehicles/doctype/vehicle_quotation/vehicle_quotation.py

```diff
@@ -66,38 +66,44 @@ def set_title(self):
 	def has_vehicle_booking_order(self):
 		return frappe.db.get_value("Vehicle Booking Order", {"vehicle_quotation": self.name, "docstatus": 1})
 
-	def update_opportunity(self):
-		if self.opportunity:
-			self.update_opportunity_status()
+	def update_opportunity(self, reopen=False):
+		if self.get("opportunity"):
+			opp = frappe.get_doc("Opportunity", self.opportunity)
+			opp.set_status(update=True, status="Open" if reopen else None)
+			opp.notify_update()
 
 	def update_lead(self):
 		if self.quotation_to == "Lead" and self.party_name:
 			doc = frappe.get_doc("Lead", self.party_name)
 			doc.set_status(update=True)
 			doc.notify_update()
 
-	def update_opportunity_status(self):
-		opp = frappe.get_doc("Opportunity", self.opportunity)
-		opp.set_status(update=True)
-		opp.notify_update()
+	@frappe.whitelist()
+	def set_is_lost(self, is_lost, lost_reasons_list=None, detailed_reason=None):
+		is_lost = cint(is_lost)
 
-	def declare_enquiry_lost(self, lost_reasons_list, detailed_reason=None):
-		if not self.has_vehicle_booking_order():
-			self.set_status(update=True, status='Lost')
+		if not is_lost and self.has_vehicle_booking_order():
+			frappe.throw(_("Cannot set as Lost as Vehicle Booking Order is made."))
```

Comment:
Improve Error Message:
Cannot declare as Lost because Vehicle Quotation is converted to order